docs/conf/portland/2018/news/writing-day.rst

```diff
@@ -0,0 +1,31 @@
+:template: {{year}}/generic.html
+
+.. post:: May 1, 2018
+   :tags: 2018, portland-2018, schedule
+
+Get Ready for Writing Day
+=========================
+
+Write the Docs is almost here. As usual, we'll kick things off with the
+`writing day <http://www.writethedocs.org/conf/portland/2018/writing-day/>`
```

Comment:
you need an underscore after the closing `\`` for the link syntax.

Reply:
So 
  
    `writing day <http://www.writethedocs.org/conf/portland/2018/writing-day/>`_ 

Throughout